examples/header-based-routing/header_based_routing.server.out

```diff
@@ -1,4 +1,5 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and use the `ballerina run` command.
-$ ballerina run header_based_routing.bal
+# `.bal` file and execute the `ballerina run` command along with the
+# Ballerina home path as a system property.
```

Comment:
I think we shouldn't say it is a 'system property'.

Reply:
Resolved